omero/developers/build-system.txt

```diff
@@ -165,6 +87,81 @@ of the :model_doc:`OME-Model <ome-xml/>` and generate our |OmeroModel|.
 This code is not available in git, but once built, can be found in all the
 directories named "generated".
 
+Build tools
+-----------
+
+Ant
+^^^
+
+``./build.py`` is a complete replacement for your local ant install. In
+many cases on and on most OS, you will be fine running :program:`ant`. If you
```

Comment:
'In many cases on and on most OS' is a bit scrambled. Suggest 'In most cases, you will...'